src/components/IPScan/Actions/Group/index.tsx

```diff
@@ -0,0 +1,172 @@
+
+import DownloadAll from './DownloadAll';
+
+import cssBinder from 'styles/cssBinder';
+
+import fonts from 'EBI-Icon-fonts/fonts.css';
+import local from '../style.css';
+
+const css = cssBinder(fonts, local);
+
+const mergeSequences = (results: Array<Iprscan5Result>) => {
+  let seqs = '';
+  let i = 1;
+  for (const result of results) {
+    seqs += `> ${result.xref?.[0]?.id || `Sequence ${i++}`}\n${(
+      (result.sequence || '').match(/(.{1,60})/g) || []
+    ).join('\n')}\n`;
+  }
+  return seqs;
+};
+
+export type Jobs = Record<
+  string,
+  {
+    metadata: IprscanMetaIDB;
+  }
+>;
+type Props = {
+  group: string;
+  deleteJob: typeof deleteJob;
+  goToCustomLocation: typeof goToCustomLocation;
+  jobs: Jobs;
+};
+
+const getJobsOfGroup = (jobs: Jobs, group: string) =>
+  Object.values(jobs).filter(({ metadata }) => metadata.group === group);
+
+export const getAllResults = async (jobs: Jobs, group: string) => {
+  const dataT = await getTableAccess(IPScanJobsData);
+  const results = [];
+  let output: Record<string, unknown> = {};
+  for (const job of getJobsOfGroup(jobs, group)) {
+    const {
+      results: r,
+      localID: _,
+      originalInput: __,
+      ...metadata
+    } = await dataT.get(job.metadata.localID);
+    results.push(r[0]);
+    output = { ...output, ...metadata };
+  }
+  output.results = results;
+  return output;
+};
+
+const getRemoteID = (jobs: Jobs, group: string) => {
+  const jobWithRemoteID = Object.values(jobs).find(
+    (job) => job.metadata.group === group && job.metadata.remoteID,
+  );
+  if (jobWithRemoteID) {
+    return jobWithRemoteID.metadata.remoteID.replace(/-\d+$/, '');
+  }
+  return;
+};
+const GroupActions = ({
+  group,
+  jobs,
+  deleteJob,
+  goToCustomLocation,
+}: Props) => {
+  const handleDelete = () => {
+    getJobsOfGroup(jobs, group).forEach((job) => deleteJob(job));
+  };
+
+  const handleReRun = async () => {
+    const results = await getAllResults(jobs, group);
+    const search: Record<string, unknown> = {};
+    if (results?.applications) {
+      search.applications =
+        typeof results.applications === 'string'
+          ? [results.applications]
+          : results.applications;
+    }
+    goToCustomLocation({
+      description: {
+        main: { key: 'search' },
+        search: {
+          type: 'sequence',
+          value: mergeSequences(results.results as Iprscan5Result[]),
+        },
+      },
+      search,
+    });
+  };
+
+  return (
+    <nav className={css('buttons')}>
+      <DropDownButton label="Group Actions" icon="&#xf03a;">
+        <ul>
+          <li>
+            <Tooltip
+              title={
+                <div>
+                  <b>Delete All Jobs</b>: This will remove the stored data of
+                  all the sequences associated with this job from your browser.
+                  Remember that search results are only retained on our servers
+                  for 7 days
```

Comment:
```suggestion
                  Delete all the stored data related to the
                  sequences associated with this job from your browser.
                  Please note that search results are stored
                  on our servers for only 7 days.
```

Also, is it relevant to mention the 7 day limit? This action clear the _local_ data...